Mage.Sets/src/mage/cards/g/GhastlyConscription.java

```diff
@@ -62,39 +61,24 @@ public GhastlyConscriptionEffect copy() {
 
     @Override
     public boolean apply(Game game, Ability source) {
-        // TODO: migrate to shared manifested code
         Player controller = game.getPlayer(source.getControllerId());
         Player targetPlayer = game.getPlayer(getTargetPointer().getFirst(game, source));
-        if (controller != null && targetPlayer != null) {
-            List<Card> cardsToManifest = new ArrayList<>();
-            for (Card card : targetPlayer.getGraveyard().getCards(StaticFilters.FILTER_CARD_CREATURE, game)) {
-                cardsToManifest.add(card);
-                controller.moveCardToExileWithInfo(card, null, "", source, game, Zone.GRAVEYARD, true);
-            }
-            if (cardsToManifest.isEmpty()) {
-                return true;
-            }
-            Collections.shuffle(cardsToManifest);
-            game.informPlayers(controller.getLogName() + " shuffles the face-down pile");
-            Ability newSource = source.copy();
-            newSource.setWorksFaceDown(true);
-            for (Card card : cardsToManifest) {
-                ManaCosts manaCosts = null;
-                if (card.isCreature(game)) {
-                    manaCosts = card.getSpellAbility() != null ? card.getSpellAbility().getManaCosts() : null;
-                    if (manaCosts == null) {
-                        manaCosts = new ManaCostsImpl<>("{0}");
-                    }
-                }
-                Card battlefieldCard = BecomesFaceDownCreatureEffect.findDefaultCardSideForFaceDown(game, card);
-                MageObjectReference objectReference = new MageObjectReference(battlefieldCard.getId(), battlefieldCard.getZoneChangeCounter(game) + 1, game);
-                game.addEffect(new BecomesFaceDownCreatureEffect(manaCosts, objectReference, Duration.Custom, FaceDownType.MANIFESTED), newSource);
-            }
-            Set<Card> toBattlefield = new LinkedHashSet();
-            toBattlefield.addAll(cardsToManifest);
-            controller.moveCards(toBattlefield, Zone.BATTLEFIELD, source, game, false, true, false, null);
+        if (controller == null || targetPlayer == null) {
+            return false;
+        }
+        List<Card> cardsToManifest = new ArrayList<>();
+        for (Card card : targetPlayer.getGraveyard().getCards(StaticFilters.FILTER_CARD_CREATURE, game)) {
+            cardsToManifest.add(card);
+            controller.moveCardToExileWithInfo(card, null, "", source, game, Zone.GRAVEYARD, true);
```

Comment:
Last param `withName` must be false to hide card name to face down in the logs

Reply:
Maybe I don't understand that param - I was thinking it should be true since moving out of a public zone - false only for moving from hidden zone to exile face down?

Reply:
Well, that's ok logic